cypress/integration/unit/helpers/string.helper.spec.ts

```diff
@@ -0,0 +1,47 @@
+/// <reference types="cypress" />
+
+import {StringHelper} from "../../../../src/helpers/string.helper";
+
+describe("StringHelper", () => {
+  describe("areEquivalent", () => {
+    it("should return true when equivalent", () => {
+      expect(StringHelper.areEquivalent("str", "str")).to.equal(true);
+      expect(StringHelper.areEquivalent("   str", "str   ")).to.equal(true);
+
+    });
+    it( "should return null when comparing null values with strings", () => {
+      expect(StringHelper.areEquivalent(null, "str")).to.equal(null);
```

Comment:
Miałem pewność, że ta metoda zawsze zwraca bool. Czy mógłbyś zamienić ją implementację na to co jest niżej i usunąć testy, w stylu `to.equal(null)` 
```ts
static areEquvivalent(string1: string | null, string2: string | null): boolean {
    return !!(
      string1 && string2 && StringHelper.getRawValue(string1) === StringHelper.getRawValue(string2)
    );
  }
```

Reply:
Zmieniłem